CONTRIBUTING.md

```diff
@@ -1,17 +1,61 @@
-Please post all questions and issues first on
-[https://discuss.elastic.co/c/beats](https://discuss.elastic.co/c/beats)
-before opening a Github Issue.
+# Contributing to Elastic Agent
 
-# Contributing to Beats
+## Got Questions?
 
-The Beats are open source and we love to receive contributions from our
-community — you!
+Have a problem you want Elastic Agent to solve for you?
 
-There are many ways to contribute, from writing tutorials or blog posts,
-improving the documentation, submitting bug reports and feature requests or
-writing code for implementing a whole new protocol.
+* You can ask a question in the [forum](https://discuss.elastic.co/c/elastic-stack/elastic-agent).
+* You are welcome to join Elastic Stack Community slack (https://elasticstack.slack.com) and ask for help on the #elastic-agent channel.
+
+## Have an Idea or Feature Request?
+
+* File an issue on [GitHub](https://github.com/elastic/elastic-agent/issues). Please remember that GitHub is used only for bugs and feature requests. If you have a general question, the [forum](https://discuss.elastic.co/c/elastic-stack/elastic-agent) or Elastic Stack Community slack (https://elasticstack.slack.com) is the best place to ask.
+
+## Something Not Working? Found a Bug?
+
+If you think you found a bug, it probably is a bug. File it in [GitHub](https://github.com/elastic/elastic-agent/issues)
```

Comment:
```suggestion
If you think you found a bug, it probably is a bug. Fill it in [GitHub](https://github.com/elastic/elastic-agent/issues)
```

Reply:
Question: Do we want to direct people to [Discuss](https://discuss.elastic.co/) like we do for Beats instead of directly opening an issue here?

Regarding the wording, I believe `File` reads better in this case.